### PR TITLE
Added Patreon and Mixcloud selectors

### PR DESCRIPTION
**Pull request by kauhat**
https://www.patreon.com/login
patreon

https://www.mixcloud.com
mixcloud

### CODE REVIEW

File: src/content_script.js [hunk 1]
```diff
@@ -40,6 +40,8 @@ const LOGIN_PATTERN_DETECTION_SELECTORS = [
   "[aria-label*='Log in with Facebook']", // Tinder
   "[action*='facebook_login']", // Airbnb
   "[action*='facebook_signup']", // Airbnb
+  "button[color='facebookBlue']", // Patreon
```
Comment (maxxcrawford) [inline src/content_script.js]: Can you add additional classes to qualify both selectors?

File: src/content_script.js [hunk 2]
```diff
@@ -40,6 +40,8 @@ const LOGIN_PATTERN_DETECTION_SELECTORS = [
...
+  "[class*='btn-facebook']", // Mixcloud
```
Comment (maxxcrawford) [inline src/content_script.js]: This selector is too broad of a class and will identify non-login items accidentally. Recommend adding some parent classes, `.modal-content`, etc.

File: src/content_script.js [hunk 3]
```diff
@@ -40,6 +40,9 @@ const LOGIN_PATTERN_DETECTION_SELECTORS = [
   "[aria-label*='Log in with Facebook']", // Tinder
   "[action*='facebook_login']", // Airbnb
   "[action*='facebook_signup']", // Airbnb
+  "[data-tag='login-form'] ~ span button[color='facebookBlue']", // Patreon
+  ".welcome-buttons .btn-facebook", // Mixcloud homepage
```
Comment (maxxcrawford) [inline src/content_script.js]: Can you include `.homepage-photo-leader` too? Also add `div` to the `.btn-facebook` class. _(Similiar to the next comment)_

File: src/content_script.js [hunk 4]
```diff
@@ -40,6 +40,9 @@ const LOGIN_PATTERN_DETECTION_SELECTORS = [
...
+  ".auth-form .btn-facebook", // Mixcloud login modal
```
Comment (maxxcrawford) [inline src/content_script.js]: Can you also add `.modal-content` to this selector? This still feels still broad. I also recommend revising the `.btn-facebook` to include the div: `div.btn-facebook`

**Comment by kauhat** [comment]
Thanks for your patience, I've updated the selectors.

**Pull request merged**